app/models/util/db_manager.rb

```diff
@@ -39,6 +39,10 @@ def dump_database
       "
       puts cmd
       run_command_line(cmd)
+      filename = File.basename(dump_file_location)
+      record = FileRecord.create(file_type: snapshot, filename: "#{filename}" ) 
```

Comment:
Hey @ievgend2 this should not have worked when you tested it. `snapshot` is passed here as a variable that has not been defined. This should be a string `"snapshot"`